packages/permissions/src/hooks/use-is-authorized/use-is-authorized.ts

```diff
@@ -70,22 +63,22 @@ const useIsAuthorized = ({
   demandedDataFences?: TDemandedDataFence[];
   selectDataFenceData?: TSelectDataFenceData;
   shouldMatchSomePermissions?: boolean;
-}) => {
+}): boolean => {
   const impliedPermissions = getImpliedPermissions(demandedPermissions);
 
-  useWarning(
```

Comment:
Is this change related to what we're experimenting with on this PR? The goal here was to show the warning only once using the `useWarning` and to not emit it on each render.

Reply:
Yes the `tiny-invariant` dep was missing, and since we prefer to use that I removed the `tiny-warning` dep.

> The goal here was to show the warning only once

Ah that's a good point, I'll change that back. Thanks!